twitter_user.go

```diff
@@ -13,9 +13,11 @@ type User struct {
 	Following                      bool     `json:"following"`
 	FriendsCount                   int      `json:"friends_count"`
 	GeoEnabled                     bool     `json:"geo_enabled"`
+	HasExtendedProfile             bool     `json:"has_extended_profile"`
```

Comment:
Picked it up from here: https://dev.twitter.com/rest/reference/get/users/show

From what I can gather from the docs, extended profiles will include extra metadata like birthday, links to Facebook / VIne / Co. You're right, Twitter is a bit sneaky with making those API changes official :(